src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -202,6 +207,7 @@ CustomFieldInputMultipleChoice.propTypes = {
 
 CustomFieldInputMultipleChoice.defaultProps = {
   errorText: undefined,
+  noOptionText: 'No options available.',
```

Comment:
It doesn't make sense to be this configurable for the multi choice component since it is always about choices.